tests/c20_client/test_client.py

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
There is a space at the top of this file. Not certain if it is a problem as static analysis does not pick it up but just wanted to ask.